docs/docs/reference/providers/overview.md

```diff
@@ -1,27 +1,7 @@
 ---
 title: Credential Providers
 id: providers
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/reference/providers/overview.html
```

Comment:
Needs a redirect added to https://docs.secretless.io/Latest/en/Content/References/providers/overview.htm

Reply:
there is one (below the comment). Is this what you meant?

Reply:
oops. I was fooled by the way gh displayed this one. thanks